src/Battlescape/TileEngine.cpp

```diff
@@ -2033,8 +2033,8 @@ bool TileEngine::tryReaction(ReactionScore *reaction, BattleUnit *target, const
 				{
 					_save->getBattleGame()->statePushBack(new ProjectileFlyBState(_save->getBattleGame(), action));
 				}
+				return true;
 			}
-			return arg.getFirst() > 0;
```

Comment:
This was deliberately done, some kind of 6th sense. To change that you need find some support that would like more your solution, e.g. Meridian, Dioxine, Solarius Scorch or others big moddes, if they prefer your version then then it will be included.